packages/server-wallet/src/wallet/store.ts

```diff
@@ -95,15 +95,6 @@ export class Store {
     await this.getOrCreateSigningAddress(new ethers.Wallet(privateKey));
   }
 
-  async getFirstParticipant(): Promise<Participant> {
-    const signingAddress = await this.getOrCreateSigningAddress();
```

Comment:
I am confused by this, because the "first" participant won't necessarily have *my* signing address.

Reply:
"first" doesn't mean first in the channel. It means first in the wallet.

Since participant management is not actually performed by the wallet, the wallet enforces that it only stores one signing key. Therefore, the "first" participant here is the only participant the wallet stores.

Moreover, as a `Participant` object, it's misleading. The participantId does not match the Id in channels that the receipt manager will join, for example: in that case, the participantId in the channel is a URL, while the Id returned by the wallet is a hex string.